LDK/resources/web/LDK/panel/TabbedReportPanel.js

```diff
@@ -31,6 +31,9 @@ Ext4.define('LDK.panel.TabbedReportPanel', {
     // Passed to filters implementing caseInsensitive id search
     caseInsensitiveSubjects: false,
 
+    // Passed to filters implementing reportQCStates filter
+    reportQCStates: false,
```

Comment:
Should this be an empty array like the other one above in the AbstractFilterType.js

Reply:
thanks fixed